torch/distributed/distributed_c10d.py

```diff
@@ -350,6 +350,18 @@ def get_global_rank(group: ProcessGroup, group_rank: int) -> int:
             return rank
     raise RuntimeError(f"Group rank {group_rank} is not part of group {group}")
 
+# TODO: remove this once the ecosystem moves away from it.
+def _get_global_rank(group, rank):
+    """
+    This method is deprecated, please use get_global_rank.
+    """
+    warnings.warn(
```

Comment:
warn only once to avoid spamming logs maybe?

Reply:
warnings.warn does that by default